src/lib/passport-jwt.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable camelcase */
+const { Strategy, ExtractJwt } = require('passport-jwt')
+const { getUserByEmail } = require('../services/users')
+
+const config = require('../config')
+
+const options = {
+	jwtFromRequest: ExtractJwt.fromAuthHeaderAsBearerToken(),
+	secretOrKey:config('TOKEN_SECRET')
+}
+
+module.exports = ( passport ) => {
+	const verifyCallBack =  async (jwt_payload, done) => {
```

Comment:
@jakazzy this doesn't have to be `jwt_payload` (snake case) since its not an object property. I'm thinking `jwtPayload` will work just fine?

 so we can avoid eslint disabling if not needed